app/views/dashboard.erb

```diff
@@ -1,21 +1,25 @@
-<div class="dashboard-container">
-  <div class="left-container"></div>
-  <div class="dashboard-center-container">
-    <div class="logo">
-      <img class="logo" src="/images/tps-logo.png" alt="tps logo">
+<header class="tps__header">
+  <img src="/images/tps-logo.png" alt="tps logo">
+</header>
```

Comment:
This same `<header>` and `<img>` combination is being used in every template, isn't it? Should it go in `layout` instead?